### PR TITLE
bugfix/EPMRPP-37573 User photo is not applied

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/controllers/user/actionCreators.js [hunk 1]
```diff
@@ -13,6 +14,12 @@ const fetchUserSuccessAction = (user) => ({
   payload: user,
 });
 
+export const setPhotoTimeStampAction = (timeStamp) => (dispatch) =>
```
Comment (DiscoElevator) [inline app/src/controllers/user/actionCreators.js]:
```
export const setPhotoTimeStampAction = (timeStamp) => ({
    type: SET_PHOTO_TIME_STAMP,
    payload: timeStamp,
  });
```

File: app/src/controllers/user/constants.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
 export const SET_ACTIVE_PROJECT = 'setActiveProject';
 export const SET_START_TIME_FORMAT = 'setStartTimeFormat';
+export const SET_PHOTO_TIME_STAMP = 'setPhotoTimeStamp';
 
 export const START_TIME_FORMAT_RELATIVE = 'relative';
 export const START_TIME_FORMAT_ABSOLUTE = 'absolute';
 
 export const SETTINGS_INITIAL_STATE = {
   startTimeFormat: START_TIME_FORMAT_RELATIVE,
+  photoTimeStamp: null,
```
Comment (DiscoElevator) [inline app/src/controllers/user/constants.js]: Maybe it should be initalized as `Date.noew()`?

Comment (AmsterGet) [reply app/src/controllers/user/constants.js]: Fixed

File: app/src/controllers/user/actionCreators.js (hunk 1 above)
Comment (AmsterGet) [reply app/src/controllers/user/actionCreators.js]: Fixed

**Pull request merged**